astropy/units/quantity_helper/erfa.py

```diff
@@ -98,6 +99,22 @@ def helper_gc2gd(f, nounit, unit1):
         )
 
 
+def helper_gc2gde(f, unit1, dimensionless_unscaled, unit2):
```

Comment:
Like below, enumerate all units, and have `get_converter(unit2, dimensionless_unscaled)` below.

By the way, feel free to give the units more meaningful names - I know I didn't do it for many functions but I changed later (see, e.g., `helper_s2pv`).